src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3794,6 +3794,9 @@ You should consider suppressing the warning only if you're sure that you don't w
   <data name="ERR_SignButNoPrivateKey" xml:space="preserve">
     <value>Key file '{0}' is missing the private key needed for signing</value>
   </data>
+  <data name="ERR_PublicSignNoKey" xml:space="preserve">
```

Comment:
It is not clear what is going on with this error message. We add it for C#, but this change never reports this error it for C#. We don't add it for VB, but do report it. And tests pass?